### PR TITLE
Zero out stats values in the cli

### PR DESCRIPTION
**Pull request by crosbymichael**
Based on some feedback, when you have a container via the cli that you
are monitoring for stats, if you stop the container it will stay in the
display but report the last datapoint that was received.

This PR changes the display to zero out the values for containers where
an update has not been received within a specified duration, i.e. 2
seconds.  This signals the user that the container has stopped as it
reports cpu and memory usage of 0.

Signed-off-by: Michael Crosby crosbymichael@gmail.com

### CODE REVIEW

File: api/client/commands.go [hunk 1]
```diff
@@ -2636,39 +2636,64 @@ type containerStats struct {
...
+	type d struct {
+		s   *stats.Stats
+		err error
+	}
 	var (
 		previousCpu    uint64
 		previousSystem uint64
 		start          = true
 		dec            = json.NewDecoder(stream)
+		updates        = make(chan *d, 1)
```
Comment (tiborvass) [inline api/client/commands.go]: for a struct of a pointer and an error interface, you don't need to make it a chan of pointer, chan of struct should be enough. So `make(chan d, 1)`.

Comment (LK4D4) [inline api/client/commands.go]: Google style guide proposing use pointers always if there are pointers fields.

Comment (crosbymichael) [inline api/client/commands.go]: Yes, that is the advice i follow

**Comment by LK4D4** [comment]
LGTM
Also I tested locally because it is too boring to write tests for such cli-things :)

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**